SeaPublicWebsite/wwwroot/css/_simple-energy-advice-custom.scss

```diff
@@ -314,7 +314,7 @@
     display: inline;
     font: inherit;
     height: auto;
-    padding: 0;
+    padding: 8px 10px 7px; /* Use same padding as govuk-button */
```

Comment:
The govuk button seems to use `padding: (govuk-spacing(2) - $govuk-border-width-form-element) govuk-spacing(2) (govuk-spacing(2) - $govuk-border-width-form-element - ($button-shadow-size / 2));`
Given that we don't have a border or shadow maybe here we can just use `padding: govuk-spacing(2)`